AlphaWallet/Tokens/ViewModels/NonFungibleTokenViewCellViewModel.swift

```diff
@@ -39,7 +41,12 @@ struct NonFungibleTokenViewCellViewModel {
     }
 
     var blockChainName: String {
-        return R.string.localizable.blockchainEthereum()
+        if chainId == 100 {
```

Comment:
I think it's better to use a switch right away. Much easier to discover this piece of code rather than remember to search for 100 later.